src/components/strategies/edit/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const getStatusTextByTxStatus = (
```

Comment:
arent we also using this for create strategy? Why have a dedicated file for this in strategies/edit folder?